feature 1443 document probabilistic field settings

georgemccabe opened the pull request:
per #1443, added information to documentation to describe how to configure METplus to read probabilistic forecast data, including examples of METplus configurations and the resulting values that are set for the fcst dictionary in the wrapped MET config file

## Pull Request Testing ##

- [X] Describe testing already performed for these changes:</br>

No tests needed

- [X] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

Review documentation additions: https://metplus.readthedocs.io/en/feature_1443_doc_prob_field/Users_Guide/systemconfiguration.html#probabilistic-forecast-fields

- [X] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**

- [X] Do these changes include sufficient testing updates? **[Yes]**

- [X] Will this PR result in changes to the test suite? **[No]**</br>

- [X] Please complete this pull request review by **3/15/2022**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [X] Review the source issue metadata (required labels, projects, and milestone).
- [X] Complete the PR definition above.
- [X] Ensure the PR title matches the feature or bugfix branch name.
- [X] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Organization** level software support **Project** or **Repository** level development cycle **Project**
Select: **Milestone** as the version that will include these changes
- [x] After submitting the PR, select **Linked issues** with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

georgemccabe committed:
per #1443, added information to documentation to describe how to configure METplus to read probabilistic forecast data, including examples of METplus configurations and the resulting values that are set for the fcst dictionary in the wrapped MET config file

georgemccabe committed:
fixed incorrect glossary term reference

georgemccabe committed:
add link for glossary term

JohnHalleyGotway commented on docs/Users_Guide/systemconfiguration.rst:
```diff
@@ -1454,6 +1454,56 @@ fcst dictionary item. If this is the case, then use these variables. If
+
+    [config]
+    FCST_IS_PROB = True
+    FCST_VAR1_NAME = APCP
```
Wondering if we should use a more likely example, like verifying the ensemble frequencies from the NetCDF output of Ensemble-Stat or Gen-Ens-Prod?
```suggestion
    FCST_VAR1_NAME = APCP_24_A24_ENS_FREQ_gt0.0
```

JohnHalleyGotway commented on docs/Users_Guide/systemconfiguration.rst:
```diff
@@ -1454,6 +1454,56 @@ fcst dictionary item. If this is the case, then use these variables. If
+
+will add the following to the MET config file::
+
+    fcst = {field = [{ name="APCP"; level="(*,*)"; prob=TRUE; cat_thresh=[ ==0.1 ]; }];}
```
If you switch, you'd need to update this line too:
```suggestion
    fcst = {field = [{ name="APCP_24_A24_ENS_FREQ_gt0.0"; level="(*,*)"; prob=TRUE; cat_thresh=[ ==0.1 ]; }];}
```

JohnHalleyGotway commented on docs/Users_Guide/systemconfiguration.rst:
```diff
@@ -1454,6 +1454,56 @@ fcst dictionary item. If this is the case, then use these variables. If
+
+    fcst = {field = [{ name="APCP"; level="(*,*)"; prob=TRUE; cat_thresh=[ ==0.1 ]; }];}
+
+The cat_thresh value used in this case defaults to ==0.1.
```
```suggestion
The cat_thresh value defaults to ==0.1 and defines the size of the Nx2 probabilistic contingency table.
```

JohnHalleyGotway reviewed (approved):
I approve of this PR. I did recommend changing the first (NetCDF) example for verifying ```APCP(*,*)``` to making it look like the NetCDF output from Ensemble-Stat ```APCP_24_A24_ENS_FREQ_gt0.0(*,*)```... or something similar.

georgemccabe committed:
Update docs/Users_Guide/systemconfiguration.rst

use a more likely example, like verifying the ensemble frequencies from the NetCDF output of Ensemble-Stat or Gen-Ens-Prod

Co-authored-by: johnhg <johnhg@ucar.edu>

georgemccabe committed:
Update docs/Users_Guide/systemconfiguration.rst

use a more likely example, like verifying the ensemble frequencies from the NetCDF output of Ensemble-Stat or Gen-Ens-Prod part 2

Co-authored-by: johnhg <johnhg@ucar.edu>

georgemccabe committed:
Update docs/Users_Guide/systemconfiguration.rst

Co-authored-by: johnhg <johnhg@ucar.edu>
